gui/editgamedialog.cpp

```diff
@@ -184,25 +184,23 @@ EditGameDialog::EditGameDialog(const Common::String &domain)
 	}
 
 	//
-	// 2) The engine tab (shown only if the engine implements one or there are custom engine options)
+	// 2) The engine's game settings (shown only if the engine implements one or there are custom engine options)
 	//
 
 	if (metaEnginePlugin) {
-		int tabId = tab->addTab(_("Engine"), "GameOptions_Engine");
+		tab->setActiveTab(0);
 
 		const MetaEngineDetection &metaEngineDetection = metaEnginePlugin->get<MetaEngineDetection>();
 		metaEngineDetection.registerDefaultSettings(_domain);
 		if (enginePlugin) {
 			enginePlugin->get<MetaEngine>().registerDefaultSettings(_domain);
-			_engineOptions = enginePlugin->get<MetaEngine>().buildEngineOptionsWidgetDynamic(tab, "GameOptions_Engine.Container", _domain);
+			_engineOptions = enginePlugin->get<MetaEngine>().buildEngineOptionsWidgetDynamic(tab, "GameOptions_Game.Container", _domain);
```

Comment:
we could even keep buildEngineOptionsWidgetDynamic using the separate Engine tab if we want, or make an additional function for that